doc/releases/changelog-dev.md

```diff
@@ -42,10 +42,33 @@
 
 <h3>Bug fixes</h3>
 
+<h3>Bug fixes</h3>
+
+* Fixes a bug in which the `expval`/`var` of a `Tensor(Observable)` would depend on the order 
+  in which the observable is defined: 
+  ```python
+  @qml.qnode(dev)
+  def circ(op):
+    qml.RX(0.12, wires=0)
+    qml.RX(1.34, wires=1)
+    qml.RX(3.67, wires=2)
+    
+    return qml.expval(op)
+  
+  op1 = qml.Identity(wires=0) @ qml.Identity(wires=1) @ qml.PauliZ(wires=2)
+  op2 = qml.PauliZ(wires=2) @ qml.Identity(wires=0) @ qml.Identity(wires=1)
+  ```
+
+  ```
+  >>> print(circ(op1), circ(op2))
+  -0.8636111153905662 0.9928086358538663
+  ```
```

Comment:
Maybe it is better to show the behaviour _after_ the bugfix? That these now return the same results?

Reply:
Agreed, was also wondering about that myself too!

Reply:
Ahh, sounds good! I guess I put the behaviour before because it was more illustrative of what the problem was. The behaviour afterwards is intuitive and expected, so its not really clear what the issue was. I will swap it over to the behaviour afterwards, and we can rely on the description above.

Reply:
Minor here: apologies, from my side I wasn't clear: I think having both before and after could be nice. Not a blocking suggestion though.